dependencies.props

```diff
@@ -6,7 +6,7 @@
   -->
   <PropertyGroup>
     <ProdConCurrentRef>7daa2e55de668c4c8ee832059aab04917153881e</ProdConCurrentRef>
```

Comment:
Is there a reason the ProdConCurrentRef didn't update?

Reply:
Yeah, known issue: https://github.com/dotnet/buildtools/issues/1990. Forgot to implement the CurrentRef update for the new build mainfest dependency info.